File: 1-Two-Sum.cpp
#define IOS ios_base::sync_with_stdio(false), cin.tie(NULL), cout.tie(0);
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        IOS
        for (int i = 0; i < nums.size() - 1; i++) {
            for (int j = i + 1; j < nums.size(); j++) {
                if (nums[i] + nums[j] == target) {
                    return {i, j};
                }
            }
        }        
        return {};
    }
};

File: 1-Two-Sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (nums, target) {
    const result = new Map();

    for (let i = 0; i < nums.length; ++i) {
        const complement = target - nums[i];

        if (result.has(complement)) {
            return [result.get(complement), i];
        } else {
            result.set(nums[i], i);
        }
    }
};